client/state/initial-state.js

```diff
@@ -48,7 +48,24 @@ function deserialize( state ) {
 	return reducer( state, { type: DESERIALIZE } );
 }
 
-function loadInitialState( initialState ) {
+function addSympathy( initialStateLoader ) {
+	if ( 'development' !== process.env.NODE_ENV || ( Math.random() > 0.5 ) ) {
+		return initialStateLoader;
+	}
+
+	console.log( 'Skipping initial state load to recreate first-load experience.' ); // eslint-disable-line no-console
+
+	try {
+		localStorage.clear();
+		indexedDB.deleteDatabase( 'calypso' );
+	} catch ( e ) {
+		// no big deal
+	}
+
+	return initialState => createReduxStore( initialState );
```

Comment:
Yes it should be `{}` if you want to ignore all state. Though I think `getInitialServerState()` is more correct here since server state should always override local values.